### PR TITLE
Adherence tab: show enrolment date for patients without BP

### PR DESCRIPTION
**Pull request by timmyjose**
Bug: https://www.pivotaltracker.com/n/projects/2184102/stories/165380393
Description: Enrolment date not shown for patients without BP readings in the Adherence Tab.

### CODE REVIEW

**Comment by divs1210** [comment]
the link to story is incorrect

**Comment by timmyjose** [comment]
> the link to pivotal story is incorrect

Thanks for catching that! Fixed with correct link to the bug in Pivotal.

**Comment by divs1210** [comment]
The adherence tab should only show patients with BPs, no?
Why are patients without BPs showing up there in the first place?

File: app/views/shared/_followup_patient_card.html.erb [hunk 1]
```diff
@@ -12,12 +12,14 @@
     <%= patient.gender.capitalize %>,
     <%= patient.current_age %>
   </div>
+  <% if show_enrolment_date %>
+   <div class="overdue-details">
```
Comment (govindkrjoshi) [inline app/views/shared/_followup_patient_card.html.erb]: Indentation is screwed up.

**Comment by timmyjose** [comment]
> Should the adherence tab should only show patients with BPs?

Based on the discussion with @dvenkatr yesterday, the expected behaviour is:
  * Show the enrolment date for all patients (with and without BP measurements), and
  * Show `Unclear` for data before the program inception date (2018-01-01).

I think the idea is to have the Adherence Tab list as is, and only fix displaying the enrolment date for all regardless of BP measurements existing or not.

File: app/views/shared/_followup_patient_card.html.erb (hunk 1 above)
Comment (timmyjose) [reply app/views/shared/_followup_patient_card.html.erb]: Fixed indentation in latest push.

**Pull request merged**